integration/update_field_compat_test.go

```diff
@@ -113,9 +113,13 @@ func TestUpdateFieldCompatMax(t *testing.T) {
 		},
 
 		"MultipleQueries": {
-			update:        bson.D{{"$max", bson.D{{"v", int32(39)}, {"a", int32(30)}}}},
+			update:        bson.D{{"$max", bson.D{{"a", int32(30)}, {"v", int32(39)}}}},
 			skipForTigris: "https://github.com/FerretDB/FerretDB/issues/1061",
 		},
+		"MultipleQueriesSorted": {
+			update: bson.D{{"$max", bson.D{{"v", int32(39)}, {"a", int32(30)}}}},
+			skip:   "https://github.com/FerretDB/FerretDB/issues/1637",
+		},
```

Comment:
Upon update, response was returned in reverse order for unset in `MultipleQueries`. For now I tag it as sorting issue and skip the test.
```
        	            	--- expected
        	            	+++ actual
        	            	@@ -3,8 +3,8 @@
        	            	     "_id",
        	            	-    "a",
        	            	-    "v"
        	            	+    "v",
        	            	+    "a"
        	            	   ],
        	            	   "_id": "unset",
        	            	-  "a": 30,
        	            	-  "v": 39
        	            	+  "v": 39,
        	            	+  "a": 30
        	            	 }
```